packages/firestore/src/platform/node_lite/connection.ts

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-import nodeFetch from 'node-fetch';
+import { fetch as nodeFetch } from 'undici';
```

Comment:
Let's rename to undiciFetch to make it crystal clear we dumped node-fetch.

Reply:
OK! Smart. Done!